docs/apis-clients/grpc.md

```diff
@@ -647,6 +677,84 @@ Returned if:
 
 - Retries is not greater than 0.
 
+## Deprecated RPCs
+
+The following RPCs are exposed by the gateway service, but have been deprecated.
```

Comment:
:wrench: We could move this section under `Technical error handling`. The error handling is more relevant than duplicated RPCs. 

At some point, it may be good to extract the RPCs into different pages.